tests/Client/UpdaterTest.php

```diff
@@ -114,10 +115,20 @@ public function testVerifiedDownload(): void
         $this->testRepo = new TestRepo($fixturesSet);
         $updater = $this->getSystemInTest();
 
-        $testFileContents = file_get_contents(__DIR__ . '/../../fixtures/TUFTestFixtureSimple/tufrepo/targets/testtarget.txt');
+        $testFilePath = realpath(__DIR__ . '/../../fixtures/TUFTestFixtureSimple/tufrepo/targets/testtarget.txt');
```

Comment:
I didn't catch this before but we should use `\Tuf\Tests\TestHelpers\UtilsTrait::getFixturesRealPath` here.